subprojects/junit5/src/main/java/org/mockito/junit5/ConfiguredWithMockito.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2018 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.junit5;
```

Comment:
Can we rename this package to `org.mockito.junit.jupiter`? The JUnit team has taken extra care not to put `5` into the package name to make major version updates possible.

Reply:
Good call, done :+1: